### PR TITLE
Configurable Updates.

### PR DESCRIPTION
**Pull request by iTitus**
Not finished yet.

### CODE REVIEW

File: Eureka [hunk 1]
```diff
@@ -1 +1 @@
-Subproject commit 5d9a0fc717b7554ddc2823743031ea620b7069a4
+Subproject commit 75a812faa337406cfb1a07563db34804011db97e
```
Comment (iTitus) [inline Eureka]: AEnterpries/Eureka@5d9a0fc717b7554ddc2823743031ea620b7069a4 doesnt even exist...
AEnterpries/Eureka@75a812faa337406cfb1a07563db34804011db97e is the newest.

Comment (AEnterprise) [inline Eureka]: git sometimes does weird things with submodules

**Comment by AEnterprise** [comment]
euh, 2 commits to change nothing? please squash this when you're done :p

**Comment by iTitus** [comment]
Of course :D

**Comment by iTitus** [comment]
Basically done, did I miss anything?

**Comment by AEnterprise** [comment]
i duno, did you also test it all?

**Comment by iTitus** [comment]
yeah, everything seems to work fine

**Comment by iTitus** [comment]
Finished and tested

**Pull request merged**